cucumber/RespostasQuestoesArgumentativas.feature

```diff
@@ -0,0 +1,88 @@
+@i9n
+
+#Controle
+
+Feature: Respostas das Quest�es Argumentativas
+    As a usu�rio esteja logado no sistema
+    I want listar e adicionar respostas e  listar quest�es
+    So that possa responder e analisar as quest�es dissertativas
+  
+Scenario: listar quest�es respondidas 
+    Given que o sistema tenha a quest�o respondida "Fa�a um cen�rio"
+    And o usu�rio esteja logado como aluno
+    When ele ver a sua lista de quest�es 
+    Then a lista ir� conter "Fa�a um cen�rio"
+
+Scenario: listar quest�es que n�o foram respondidas  
+    Given que o sistema tenha a quest�es n�o respondida "Fa�a uma feature"
+    And o usu�rio esteja logado como aluno
+    When ele ver a sua lista de quest�es 
+    Then a lista ir� conter "Fa�a uma feature"
+
+Scenario: responder logado como professor
```

Comment:
Se existem os papéis de professor e aluno, existe diferença de uso do sistema entre eles. Talvez fosse mais adequado separar em arquivos de feature diferente. Na parte da descrição da feature, inclusive, deixaria claro o papel e o que se espera que seja possível realizar.